src/cplscheme/MultiCouplingScheme.cpp

```diff
@@ -35,7 +36,7 @@ MultiCouplingScheme::MultiCouplingScheme(
       _m2ns(std::move(m2ns)), _controller(controller), _isController(controller == localParticipant)
 {
   PRECICE_ASSERT(isImplicitCouplingScheme(), "MultiCouplingScheme is always Implicit.");
-
+  PRECICE_CHECK((!_isController and ((not utils::MasterSlave::isMaster()) and (not utils::MasterSlave::isSlave())) or _isController), "Non-controller participants in the MultiCouplingScheme cannot be run in parallel. Either run participant {} in serial or change the controller to another participant.", localParticipant);
```

Comment:
It's a bit more complicated than this. We cannot run those participants in parallel that need to receive a mesh that has no geometric association to any provided mesh. So the participant that has a `use-mesh from="OtherPart"` tag.
This could be the controller or any other participant.
Mmh, maybe check if there is already another error message that we would anyway run into.

Reply:
We already hit an error in `ReceivedPartition` that the mesh needs a mapping. I am not sure how to check what you described actually. We can do it on the configuration level but I am not sure. Maybe for the time being we do not allow the parallel cases (for both controller and non-controller) and once the PR of David is done we can come back to this?

Reply:
> I am not sure how to check what you described actually.

Me neither :grin: 

> Maybe for the time being we do not allow the parallel cases (for both controller and non-controller)

I thought about this too. Problem is that this would forbid cases that were allowed previously (before this PR).

Maybe we can just remove this one here and just let the user run into the "needs a mapping" error for the moment.